core.js

```diff
@@ -217,6 +217,7 @@ function processItemAsync(runner, currentValidation, key, context) {
 function addError(errors, key, validation) {
   return function(err) {
     var fieldError = errors[key];
+    err.message = validation.message || err.message;
```

Comment:
I just refined the tests to check for validation-fields with one rule and with several rules. If I try the version without modifying `err`, it's not passing. I can make another PR for those test-refinements, if this PR should be merged.